spec/build_spec.lua

```diff
@@ -132,21 +129,21 @@ describe("LuaRocks build tests #integration", function()
    end)
 
    describe("LuaRocks build - basic builds", function()
-      it("LuaRocks build luadoc", function()
+      pending("LuaRocks build luadoc", function()
          assert.is_true(run.luarocks_bool("build luadoc"))
       end)
       
-      it("LuaRocks build luacov diff version", function()
+      pending("LuaRocks build luacov diff version", function()
```

Comment:
this one (or preferrably something like this using a fixture) is worth keeping because it tests `luarocks build` receiving a version number as a second argument.